Sources/PayPalUI/PaymentButton.swift

```diff
@@ -4,7 +4,11 @@ import UIKit
 public class PaymentButton: UIButton {
 
     // asset identifier path for image and color button assets
+    #if SWIFT_PACKAGE
```

Comment:
Just a general question but can we link in the PR description the issue and where the solution was found? More-so just curious and would be great to have it called out somewhere incase we run across a similar thing in the future!

Reply:
@jaxdesmarais i need to create a ticket for it! 
GPS folks wanted to test the sdk, so they simply imported it with SPM and told me that the buttons werent working (you could only see a yellow square for example). 
Adding `Boundle.module` was found in stackoverflow and the apple docs
The idea of puting `#if` was @scannillo !
i'll put more details in the ticket

Reply:
Awesome! Thanks @jcnoriega!